yaml/boundaries.yaml

```diff
@@ -123,6 +123,19 @@ filters:
   #    `unrecognized_*` (where XX is the viewpoint / country code and * is the
   #    kind of boundary it is).
   #
+
+  # Additionally, we filter out disputes that are not currently desired for widespread display.
+  # These get a kind of `unrecognized` in order to suppress their display
+  # These have been marked in the lua with unrecognized_dispute=yes to identify them.
+  - filter:
+      all:
+        - geom_type: line
+        - unrecognized_dispute: yes
+    min_zoom: 8
```

Comment:
This is OK to be 8 here, because it for OSM features which generally come in at zoom 8. The Natural Earth features separately come in earlier with per feature min_zooms.